### PR TITLE
docs: node-first readme

### PR DESCRIPTION
**Pull request by harrysolovay**

### CODE REVIEW

File: Readme.md [hunk 1]
```diff
@@ -14,49 +14,38 @@ interactions without compromising either performance or ease of use.
 - [API Reference &rarr;](https://deno.land/x/capi/mod.ts)<br />A generated API
   reference, based on type signatures and TSDocs.
 
-## At a Glance
+## Installation
 
-Run the Capi development server.
+We can substitute the URL's target (in this case the WebSocket URL of a proxy).
```
Comment (tjjfvi) [inline Readme.md:19]: What does this mean?

File: Readme.md [hunk 2]
```diff
@@ -14,49 +14,38 @@ interactions without compromising either performance or ease of use.
...
+### [Deno](https://deno.land/)
 
-```json
-{
-  "imports": {
-    "@capi/polkadot/": "http://localhost:4646/frame/wss/rpc.polkadot.io/@latest/"
-  }
-}
+```ts
+import {} from "https://capi.dev/frame/wss/rpc.polkadot.io/@latest/mod.js"
```
Comment (tjjfvi) [inline Readme.md:30]: This should use the import map IMO

File: Readme.md (hunk 1 above)
Comment (harrysolovay) [reply Readme.md:19]: Meaning that it can be swapped out with another URL / target

File: Readme.md (hunk 2 above)
Comment (harrysolovay) [reply Readme.md:28]: Good point

Comment (harrysolovay) [reply Readme.md:28]: Actually, that isn't the case (bc the extension).

Comment (tjjfvi) [reply Readme.md:28]: True. Still, I think it's cleaner as an import map – we want our readme to show the expected use.

Comment (harrysolovay) [reply Readme.md:28]: Sounds good. In that case, is it worth import-mapping directly to the `mod.js`? That way we can keep it extension-free after all.

Comment (harrysolovay) [reply Readme.md:28]: Or I suppose this isn't the intended use, as we want people to be able to easily access files within the codegened dir (such as guards, factories, misc.)

File: Readme.md [hunk 3]
```diff
@@ -14,49 +14,44 @@ interactions without compromising either performance or ease of use.
...
 {
   "imports": {
-    "@capi/polkadot/": "http://localhost:4646/frame/wss/rpc.polkadot.io/@latest/"
+    "@capi/polkadot": "http://localhost:4646/frame/wss/rpc.polkadot.io/@latest/"
   }
 }
 ```
 
-Then, open your editor and import from the mapped chain module.
+## At A Glance
```
Comment (tjjfvi) [inline Readme.md:37]:
```suggestion
## At a Glance
```

**Review by tjjfvi — APPROVED** [review]

**Pull request merged**